apps/wa/wa-task-batch-replication-check/prod-01.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-replication-check
+spec:
+  values:
+    job:
+      schedule: "0,10,20,30,40,50 * * * *"
+      environment:
+        JOB_NAME: PERFORM_REPLICATION_CHECK
```

Comment:
you could remove duplication of environment.JOB_NAMe by moving it to its base file?  https://github.com/hmcts/cnp-flux-config/blob/88a538dbd394b0cf7126cb919bc452bc962961bd/apps/wa/wa-task-batch-replication-check/wa-task-batch-replication-check.yaml

Reply:
Hi @thomast1906 - not sure if @rasikasri is still available or has access as he is leaving.  Do you mean adding to the base kustomization.yaml file?

The JOB_NAME is used in various different CronJobs we have which have different names so if I understand correctly the above would still have to be overridden here even if we put a default in the base kustomization.

Reply:
Actually ignore that I guess you mean this base file: wa-task-batch-replication-check.yaml
and adding:

```
      environment:
        JOB_NAME: PERFORM_REPLICATION_CHECK
```

Reply:
Done in the latest commit